src/statecharts/scheduler.cljc

```diff
@@ -0,0 +1,46 @@
+(ns statecharts.scheduler
```

Comment:
This namespace was separated from `statetcharts.delayed` to avoid a circular dependency (of namespaces) between `store`, `impl` and `delayed`. I'm not sure it's the best way to organize things—I don't like that `make-scheduler` has changed namespaces—but it's the first thing I got working.